src/store/journal/__tests__/journal.effects.spec.ts

```diff
@@ -96,7 +96,7 @@ describe('Journal Effects', () => {
     // ASSERT
     effects.loadJournal$.subscribe((result) => {
       expect(journalProvider.getJournal).toHaveBeenCalled();
-      // TODO: Reinstate when offline functionality is done
+      // @TODO MES-7134: Reinstate when offline functionality is done
```

Comment:
can you enable this line and remove the TODO - the test passes

Reply:
removed